src/lib/components/Card/Card.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+  import "./Card.scss";
+  import Button from "$lib/components/Button";
+  import Image from "$lib/components/Image";
+  import classNames from "$lib/util/classNames";
+  import type { Variant } from "./cardOptions";
+
+  export let variant: Variant = "primary";
+
+  const variantClassesDict: Record<Variant, string> = {
+    primary: $$slots.image ? "usa-card--flag" : "",
+    secondary: "",
+  };
+  $: variantClass = variantClassesDict[variant];
+
+  let className = "";
+  export { className as class };
+  $: classes = classNames("usa-card", variantClass, className);
+</script>
+
+<li class={`${classes}`}>
+  <div class="usa-card__container">
+    {#if $$slots.header}
+      <div class="usa-card__header">
+        <slot name="header">
+          <h2 class="usa-card__heading">Default header</h2>
```

Comment:
Do we need these defaults if we're always checking that a slot is provided before rendering? Or is the point here to show an example of the slot content?

Reply:
that's a good catch. I think this was before I started utilizing the `$$slots` argument, so this default will never actually render in the current code.